Assets/Scripts/Background/BeyondBorderResize.cs

```diff
@@ -0,0 +1,118 @@
+// van den Ouweland, L. (2019, October 9). STRETCH A UNITY SPRITE TO FILL THE SCREEN IN A 2D GAME (GAMEOBJECT, NOT UI-CANVAS). 
+// Loek van den Ouweland Software Engineering. Retrieved March 20, 2023, from 
+// https://www.loekvandenouweland.com/content/stretch-unity-sprite-to-fill-the-screen.html.
+// This code increases/decreases the size of the GameObject sprite so that it can cover the empty left/right background space that exists 
+// when the player is using a large/horizontal screen to play the game. The sizing is dependent on the distance between the assigned border's 
+// corresponding side (boolean right) and the same camera border side. NOTE: This script must be applied to 2 filler sprites (right and left) to 
+// fill all empty space to the left and right of the game screen. This script was created for tiled continuous sprites in mind, which is why 
+// the size of the sprite is altered, not the scale.
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BeyondBorderResize : MonoBehaviour
+{
+    private float cameraHeight;
+    private float cameraWidth;
+    private float spriteWidth;
+    private float spriteHeight;
+    private float sizeFactorWidth;
+    private float sizeFactorHeight;
+    private float borderWidth;
+    private float borderEdge;
+    private float cameraEdge;
+    private float localXScale;
+    private Vector2 currentSize;
+    private SpriteRenderer fillerSprite;
+
+    public bool right;  // true if filling the space for the right side of the screen, false if otherwise
+    public SpriteRenderer border;  // variable to store border sprite
+    
+    
+    void Awake()
+    {
+        cameraHeight = Camera.main.orthographicSize * 2;
+        cameraWidth = cameraHeight * Camera.main.aspect;
+        currentSize = new Vector2(cameraWidth, cameraHeight);
+        localXScale = transform.localScale.x;
+        borderWidth = border.size.x * border.transform.localScale.x;
+
+        if (!right) {
+            borderEdge = border.transform.position.x - (borderWidth / 2);
+            cameraEdge = Camera.main.transform.position.x - (cameraWidth / 2);
+        } else {
+            borderEdge = border.transform.position.x + (borderWidth / 2);
+            cameraEdge = Camera.main.transform.position.x + (cameraWidth / 2);
+        }
+        
+        fillerSprite = GetComponent<SpriteRenderer>();
+
+        // If background + edges combined are already wider than the camera, no need to resize sprite.
+        if ((right && borderEdge < cameraEdge) || (!right && borderEdge > cameraEdge)) {
+            spriteWidth = fillerSprite.size.x;
+            spriteHeight = fillerSprite.size.y;
+
+            // The sizing factor is calculated, with the sprite renderer transformed to fill an empty side of the screen.
+            if (!right) {
+                sizeFactorWidth = Mathf.Abs(borderEdge - cameraEdge) / spriteWidth;

```

Comment:
`/ spriteWidth` is unnecessary because `sizeFactorWidth` is multiplied by `spriteWidth` when it's used below. `/ spriteHeight` when settting `sizeFactorHeight` is also unnecessary because `sizeFactorHeight` is multiplied by `spriteHeight` when it's used below.

Reply:
Got it, "spriteWidth" and "spriteHeight" have both been removed, since they are unused beyond the unneeded multiplication and division mentioned above.